src/Atoms/GreyLabel/GreyLabelStyled.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import FONTS from '../../Styles/fontsStyledComponents';
+import COLORS from '../../Styles/varsStyledComponents';
+
+const GreyLabelContainer = styled.div`
+  background: ${COLORS.grey00};
+  height: 2.5rem;
+  font-family: Mulish;
```

Comment:
You can erase this line - we had this in our index.css at this moment 
`* {
  font-family: 'Mulish', sans-serif
}`